demo/App.tsx

```diff
@@ -51,53 +61,91 @@ const App = () => {
             <View
               style={{
                 width: '45%',
-                backgroundColor: 'skyblue',
+                marginBottom: 15
+              }}>
+              <Button
+                color="red"
+                testID="runNativeError"
+                accessibilityLabel="runNativeError"
```

Comment:
The `accessibilityLabel` read-out to users who use Voice over services. So I would expect the value to be more like the `title`. 

Example: `Run native error`